source/databases-collection.txt

```diff
@@ -250,21 +250,15 @@ within 35 milliseconds of the closest member's ping time.
    :manual:`localThreshold </reference/program/mongos/#std-option-mongos.--localThreshold>`
    command-line option.
 
-.. TODO: 
```

Comment:
Is a trouble shooting section no longer necessary? This todo could be left in if the section is needed.

Reply:
We don't have a troubleshooting section on any other Databases & Collections pages across drivers, so I didn't think it was relevant - i can still leave the TODO if you think we should add it in the future

Reply:
Got it; if it's not on the other pages, seems fine to leave out!